llms/mlx_lm/utils.py

```diff
@@ -436,17 +436,25 @@ def upload_to_hub(path: str, upload_repo: str, hf_path: str):
 
     from . import __version__
 
+    api = HfApi()
+    user_info = api.whoami()
+    fullname = user_info["fullname"]
+    name = user_info["name"]
+
     card = ModelCard.load(hf_path)
     card.data.tags = ["mlx"] if card.data.tags is None else card.data.tags + ["mlx"]
     card.text = dedent(
         f"""
         # {upload_repo}
-        This model was converted to MLX format from [`{hf_path}`]() using mlx-lm version **{__version__}**.
-        Refer to the [original model card](https://huggingface.co/{hf_path}) for more details on the model.
+        
+        The Model [{upload_repo}](https://huggingface.co/{upload_repo}) was converted to MLX format from [{hf_path}](https://huggingface.co/{hf_path}) using mlx-lm version **{__version__}**.
+        
+        Model added by [{fullname}](https://huggingface.co/{name}).
```

Comment:
I'm not against it.. but I'm also not sure everyone will want their username by default in the model care 🤔

Reply:
Initially, I wasn't sure about this either. However, when I saw people adding their personal information to the model cards (e.g., [mlx-community/dbrx-instruct-4bit](https://huggingface.co/mlx-community/dbrx-instruct-4bit), [mlx-community/gemma-1.1-2b-it-4bit](https://huggingface.co/mlx-community/gemma-1.1-2b-it-4bit), etc.), I felt that they should be proud of uploading their models. At the same time, when I use models from the mlx community, I sometimes want to know who contributed to them.

Of course, some people might not want to display their information. This adjustment is not a must for me, and if needed, I can remove that part of the content.

Reply:
It's totally ok if people want to add their contact info in the cards for models they upload, but I prefer not to make it the default behavior as it may surprise people who are just experimenting or something like that.